rts/lra/lra-client/pom.xml

```diff
@@ -57,6 +57,16 @@
             <version>2.5</version>
             <scope>provided</scope>
         </dependency>
+      <dependency>
+        <groupId>org.eclipse.microprofile.rest.client</groupId>
+        <artifactId>microprofile-rest-client-api</artifactId>
+        <version>1.4.1</version>
```

Comment:
Do you think we can move the version definition to a maven property?

Reply:
sure, that is an oversight on my side, thanks